### ISSUE TITLE
Things should be read that aren't

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/friction#269]
**Test device**
Dell
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/717.
The about dialog no longer reads out items when clicked on. They were read out in the published version of JT. In addition, the Physics and Chemistry labels on the testbooks aren't read out when clicked on.

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Friction‬
URL: https://phet-dev.colorado.edu/html/friction/1.6.0-dev.19/phet/friction_all_phet.html
Version: 1.6.0-dev.19 2021-10-08 22:59:15 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.54 Safari/537.36
Language: en-US
Window: 1280x649
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/joist/issues/763 was done and is ready for review, this should no longer be an issue for Friction.

**Comment by zepumph** [comment]
Thanks! Removing blocking label, as things are working well in friction.

**Comment by Nancy-Salpepi** [comment]
Looks good on master.
